### PR TITLE
Formatted the Notebook according to the new template

### PR DESCRIPTION
**Pull request by amarlearning**
Formatted the Notebook according to the new template

### CODE REVIEW

File: doc/ipython-notebooks/classification/HashedDocDotFeatures.ipynb [hunk 1]
```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### by Evangelos Anagnostopoulos (GitHub ID: <a href=\"http://github.com/van51\">van51</a>).<br> Special thanks to my mentors for this project on GSoC 2013 : Soeren Sonnenburg, Olivier Chapelle and Benoit Rostykus</br>"
+    "#### By Evangelos Anagnostopoulos (GitHub ID: <a href=\"http://github.com/van51\">van51</a>)"
```
Comment (karlnapf) [inline doc/ipython-notebooks/classification/HashedDocDotFeatures.ipynb]: no more special thanks?

**Comment by karlnapf** [comment]
Nice thanks.
Can you post a gist of the notebook so I can see how it looks?

**Comment by amarlearning** [comment]
@karlnapf why to post a gist, you can view it on my branch [here](https://github.com/amarlearning/shogun/blob/cfabbf5be91a0bf3d5d776b92096695695a7467d/doc/ipython-notebooks/classification/HashedDocDotFeatures.ipynb)

**Comment by karlnapf** [comment]
Sure, in this case that's ok since no outputs have been changed. My bad :)

**Comment by amarlearning** [comment]
@karlnapf let me know if you want any more changes :smile:

File: doc/ipython-notebooks/classification/HashedDocDotFeatures.ipynb (hunk 1 above)
Comment (amarlearning) [inline doc/ipython-notebooks/classification/HashedDocDotFeatures.ipynb]: @karlnapf  That's wasn't much useful information! Your call, if you say I can add that again ?
But this looks good :)

Comment (amarlearning) [inline doc/ipython-notebooks/classification/HashedDocDotFeatures.ipynb]: @karlnapf Do you want anymore changes ?

Comment (karlnapf) [inline doc/ipython-notebooks/classification/HashedDocDotFeatures.ipynb]: Yes, read above

Comment (amarlearning) [inline doc/ipython-notebooks/classification/HashedDocDotFeatures.ipynb]: @karlnapf Is there any particular place where you want me to mention that .?
According to me, this should be mentioned at the end of document.

Comment (karlnapf) [inline doc/ipython-notebooks/classification/HashedDocDotFeatures.ipynb]: Sure, or in the first cell that is not used for description

**Comment by amarlearning** [comment]
@karlnapf changes made. Please review it!

**Comment by karlnapf** [comment]
Can you please squash the commits?
See our new DEVELOPING.md readme in the wiki for how our process works
Then ready to merge

**Comment by amarlearning** [comment]
@karlnapf Again not able to squash the commit `+` did something terrible!

**Comment by karlnapf** [comment]
Please dont close pull reqeusts and then open other ones. You can always update a PR

**Comment by karlnapf** [comment]
The only way around "doing something terrible" is to read the docs.
Check our develoing.md readme in the wiki for some starting points on how to use git

**Comment by amarlearning** [comment]
@karlnapf This time I have deleted this repository and cloned a new one.
I would request you to consider this [PR](https://github.com/shogun-toolbox/shogun/pull/3569) for now. Next time I'll take care of that.

**Comment by karlnapf** [comment]
Ok sure. 
In the future **dont** delete and re-create your fork. It is not necessary and the worst approach to "fixing" git problems. Just read the git manuals...

**Pull request closed without merging**